feature-governance-impl/src/main/java/io/novafoundation/nova/feature_governance_impl/presentation/tindergov/basket/TinderGovBasketViewModel.kt

```diff
@@ -140,16 +143,23 @@ class TinderGovBasketViewModel(
             val availableToVoteReferenda = availableToVoteReferendaFlow.first()
 
             val removedReferenda = basket.filter { availableToVoteReferenda.contains(it.referendumId).not() }
+                .filter { interactor.basketItemValidForVote(it) }
```

Comment:
This is strange. Why does removed referenda is filtered list is by the validity?  Even if its correct (e.g. if you aim to filter out "invalid by voting power" referenda and only show number "invalid by status" this code doesnt make sense for the external reader who might not know the context